docs/getting-started/install-endpoint.asciidoc

```diff
@@ -92,60 +92,88 @@ If the prompt does not appear because you're using a version earlier than macOS
 
-To access the security integration policy:
+To access the Integration Policy:
 
 1. In the {security-app}, select the **Administration** tab to view the Endpoints list. Remember that you must have admin permissions in {kib} to access this page.
-2. From the **Integration Policy** column, select the Policy you want to configure. The Integration Policy page appears.
+2. From the **Integration Policy** column, select the Policy you want to configure. The Integration Policy Configuration page appears.
 
 [discrete]
 [[malware-protection]]
 === Malware protection
 
-By default, the **Malware Protections Enabled** toggle is on, with host notifications enabled or disabled based on the protection level. To disable malware protection, switch the toggle off. Malware protection levels are as follows:
+Malware prevention on the {agent} detects and stops malicious attacks by using a <<machine-learning-model, machine learning model>> that looks for static attributes to determine if a file is malicious or benign.
+
+By default, malware protection is enabled on Windows and macOS hosts. To disable malware protection, switch the **Malware protections enabled** toggle off. Malware protection levels are as follows:
 
 * **Detect**: Detects malware on the host and generates an alert. The agent will **not** block malware. You must pay attention to and analyze any malware alerts that are generated. Notifications do not appear by default. Select the **Notify User** option to enable them.
 * **Prevent** (Default): Detects malware on the host, blocks it from executing, and generates an alert. Notifications appear by default. Deselect the **Notify User** option to disable them.
 +
-TIP: Platinum and enterprise users can customize these notifications using the `Elastic Security {action} {filename}` syntax. 
+TIP: Platinum and Enterprise customers can customize these notifications using the `Elastic Security {action} {filename}` syntax.
 
+[role="screenshot"]
+image::images/install-endpoint/malware-protection.png[]
+
+[discrete]
+[[ransomware-protection]]
+=== Ransomware protection
+
+Behavioral ransomware prevention on the Elastic Agent detects and stops ransomware attacks on Windows systems by analyzing data from low-level system processes, and is effective across an array of widespread ransomware families — including those targeting the system’s master boot record.
+
+Ransomware protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum+ license from Basic or Gold, ransomware protection will be disabled by default.
+
+Ransomware protection levels are as follows:
+
+* **Detect**: Detects ransomware on the host and generates an alert. The agent will **not** block malware. Select the **Notify User** option to enable user notifications.
```

Comment:
Unsure whether "agent" should be changed to "Elastic Agent". With the exception of a few instances, most other mentions in this file use the full name of the agent. Also wanted to check whether you meant to bold "not" for emphasis or italicize it.